CONTRIBUTING.md

```diff
@@ -127,6 +127,28 @@ done the following things first:
    the work by you, as defined in the Apache 2.0 license, shall be dual
    licensed as above, without any additional terms or conditions.
 
+To streamline the process, the `.pre-commit.sh` script will run tests on each
+commit in a sanitary way for you; to have it run automatically on each commit
+simply create a symbolic link with
+
+```bash
+$ ln -s -f ../../.pre-commit.sh .git/hooks/pre-commit
+```
+
+This ensures that you can't commit your changes if tests fail.  If you need to
+make a commit without running tests, then simply pass the `--no-verify` flag.
+We recommend the following aliases:
+
+```bash
```

Comment:
Wouldn't necessarily recommend these in a `CONTRIBUTING.md`; users not familiar to `git` aliases won't know what they did and how to commit.